cfme/tests/containers/test_static_custom_attributes.py

```diff
@@ -63,6 +64,7 @@ def test_edit_static_custom_attributes(provider):
         * The attributes was successfully updated to the new values
     """
 
+    provider.add_custom_attributes(*ATTRIBUTES_DATASET)
     edited_attribs = ATTRIBUTES_DATASET
     for ii, value in enumerate(VALUE_UPDATES):
         edited_attribs[ii].value = value
```

Comment:
IIUC this mutates ATTRIBUTES_DATASET and might leak other test cases.
could `copy.deep_copy` it first, or have a 2nd EDITED_DATASET const...